modules/smart-agent_kubernetes-workloads-count/variables-gen.tf

```diff
@@ -0,0 +1,62 @@
+# workloads_count detector
+
+variable "workloads_count_notifications" {
+  description = "Notification recipients list per severity overridden for workloads_count detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "workloads_count_aggregation_function" {
+  description = "Aggregation function and group by for workloads_count detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".sum(by=['kubernetes_name','kubernetes_namespace']).count()"
+}
+
+variable "workloads_count_transformation_function" {
+  description = "Transformation function for workloads_count detector (i.e. \".mean(over='5m')\")"
+  type        = string
+  default     = ".min(over='30m')"
+}
+
+variable "workloads_count_tip" {
+  description = "Suggested first course of action or any note useful for incident handling"
+  type        = string
+  default     = ""
+}
+
+variable "workloads_count_runbook_url" {
+  description = "URL like SignalFx dashboard or wiki page which can help to troubleshoot the incident cause"
+  type        = string
+  default     = ""
+}
+
+variable "workloads_count_disabled" {
+  description = "Disable all alerting rules for workloads_count detector"
+  type        = bool
+  default     = null
+}
+
+variable "workloads_count_disabled_critical" {
+  description = "Disable critical alerting rule for workloads_count detector"
+  type        = bool
+  default     = null
+}
+
+variable "workloads_count_disabled_major" {
+  description = "Disable major alerting rule for workloads_count detector"
+  type        = bool
+  default     = null
+}
+
+variable "workloads_count_threshold_critical" {
+  description = "Critical threshold for workloads_count detector in entries"
+  type        = number
+  default     = 20
```

Comment:
may be it could be safer to not set a default, this will force the user to define an appropriate value given that is is custom in essence.